dev/ci/ci-common.sh

```diff
@@ -19,26 +22,19 @@ then
 elif [ -d "$PWD/_build_vo/" ];
 then
     # Dune Ocaml build, vo build using make
```

Comment:
It could be worth to echo something BTW so that developers switching the two modes understand how the choice was made.

Reply:
Good idea.

Reply:
I see we already echo a lot of stuff tho, including this info. So if we want to clean that I'm fine but as of now the message wouldn't provide a lot of info IMO unless we make the output cleaner.

Reply:
Maybe we should set -x only on the CI, locally it's not so useful.

Reply:
Yup; even in the CI I'd prefer a silent setup [and a full log as an artifact] , but I have very low visual tolerance for noisy builds